.kokoro/continuous/node10/docs.cfg

```diff
@@ -1,4 +1,4 @@
 env_vars: {
     key: "TRAMPOLINE_BUILD_FILE"
-    value: "github/nodejs-appengine-admin/.kokoro/docs.sh"
+    value: "github/nodejs-appengine/.kokoro/docs.sh"
```

Comment:
@bcoe @sofisl Where did the `-admin` originally come from?

Looking at my few changes to `owlbot.py`, I don't see why the output differs.  Do you?

Reply:
@SurferJeffAtGoogle,  I think we needed to merge #29 to fix this issue. Can you try regenerating and see if the issue still appears?

Reply:
I regenerated, and now I only see the -admin suffix being dropped in the system test.